lms/static/js/student_account/views/LoginView.js

```diff
@@ -102,7 +103,11 @@
             },
 
             saveError: function( error ) {
-                this.errors = ['<li>' + error.responseText + '</li>'];
+                if(error.status == 500){
```

Comment:
Use ```===``` and ```!==``` over ```==``` and ```!=```.

Ref: [airbnb guide # 15.1](https://github.com/airbnb/javascript#15.1)